src/_data/styles/gradientcards.json

```
@@ -0,0 +1,5 @@
+{
+  "title": "Gradient Cards",
+  "author": "Bente Hinkenhuis",
+  "stylesheet": "https://raw.githubusercontent.com/Siggi005/Stylestage/master/style.css",
```

```suggestion
  "stylesheet": "https://raw.githubusercontent.com/Siggi005/Stylestage/master/style.css"
```

Extra comma invalidates JSON :)